### DISCUSSION TITLE
Session switching / multiple session cookies

### DISCUSSION THREAD

**Discussion by theogravity** [ory/hydra#3119] [Q&A] [answered]
We're in the process of evaluating alternative auth systems such as Ory, and we have an important use-case that I cannot seem to find the answer to in the documentation.

We have a web-facing app where the user may have multiple accounts and the user can freely switch between them, akin to the Google account switcher. This isn't a multi-tenant scenario, just a user may have multiple accounts with our app and we'd like them to be able to sign in and switch between them freely. They may be using a social login or an email / pass login.

Does Ory support this kind of mechanism when it comes to a web-facing app? For native app, I don't think this is an issue as we can store multiple session tokens and reference them from there, but for web-facing, it seems there is a single Ory session cookie based on what I've read.

**Comment by aeneasr** [comment, marked as answer]
Since you control the login app, you can tell Hydra which user is signed in! As long as your login backend supports it, Hydra does too!

**Reply by theogravity** [reply]
> Since you control the login app, you can tell Hydra which user is signed in! As long as your login backend supports it, Hydra does too!

Would I be managing several session cookies in this case?

**Reply by aeneasr** [reply]
exactly
